service/geopm-service.spec.in

```diff
@@ -38,59 +41,98 @@ Vendor: Intel Corporation
 URL: https://geopm.github.io
 Source0: geopm-service.tar.gz
 BuildRoot: %{_tmppath}/geopm-service-%{version}-%{release}-root
+Prefix: %{_prefix}
+
 BuildRequires: gcc-c++
 BuildRequires: unzip
 BuildRequires: libtool
 BuildRequires: systemd-devel
```

Comment:
I guess we require version 221 or higher, for the sd-bus interface. If our supported OSs all have at least that version, then I guess the version annotation is not needed in this file. If not, maybe clarify the note in https://geopm.github.io/requires.html#systemd-library-requirement to specify why we require newer distributions for the service.